src/app/pages/common/entity/entity-wizard/entity-wizard.component.ts

```diff
@@ -234,7 +234,7 @@ export class EntityWizardComponent implements OnInit {
     }
   }
 
-  selectionChange(event) {
+  selectionChange(event: any) {
```

Comment:
`event` here should be of type `StepperSelectionEvent`

Reply:
Updated